app/src/main/res/layout/fragment_help_category.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <variable
+            name="item"
+            type="cz.covid19cz.erouska.ui.help.data.Question" />
+
+        <variable
+            name="vm"
+            type="cz.covid19cz.erouska.ui.helpcategory.HelpCategoryVM" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingHorizontal="@dimen/fragment_padding">
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/help_questions"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:onClick="@{ () -> vm.onItemClicked(item)}"
+            app:items="@{ vm.items }"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            app:layoutStrategy="@{vm.layoutStrategy}"
+            app:layout_constraintStart_toStartOf="parent"
+            app:viewModel="@{ vm }" />
+
+        <View
```

Comment:
why do you use here for **divider** this view with the background when you use in `item_help_question.xml`
```
<View
     android:id="@+id/divider_top"
     style="@style/Divider"
```

please, unify it (with @style/Divider)